src/ConcreteOperations/interior.jl

```diff
@@ -0,0 +1,28 @@
+export is_interior_point
+
+
+"""
+    is_interior_point(d::AbstractVector, P::LazySet; p=Inf, ε=_TOL_F64.rtol)
```

Comment:
```suggestion
    is_interior_point(d::AbstractVector, P::LazySet; p=Inf, ε=_rtol(N))
```

Reply:
Same thing: copy the whole signature here.